core/src/main/resources/plugins_stream_support.txt

```diff
@@ -16,4 +16,5 @@ kotlin=1.1
 ruby=1.1
 sonarscala=1.4
 tsql=1.1
-xml=2.0.1
\ No newline at end of file
+xml=2.0.1
+go=1.6
```

Comment:
just realized that there is two times XML in this file.

Reply:
That's the benefit of lexicographic sorting like in the other file ;)